src/index.html

```diff
@@ -1,13 +1,474 @@
+                    <a
+                      href="https://goo.gl/maps/2s3KcERf5WhcQ5N16"
+                      target="_blank"
+                      rel="noopener"
+                    >
+                      <img
+                        src="./images/photos/4.jpg"
+                        alt="Place 4"
+                        class="place__photo"
+                      >
+                    </a>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                href="https://goo.gl/maps/2s3KcERf5WhcQ5N16"
+                class="place__link"
+                target="_blank"
+                rel="noopener"
+              >
+                Edition Residens
+              </a>
+              / Miami Beach, FL 33139
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4">
+                  <div class="place__photo-container place__photo-container--left">
+                    <a
+                      href="https://goo.gl/maps/jUJWjAFsmLsUbAch9"
+                      target="_blank"
+                      rel="noopener"
+                    >
+                      <img
+                        src="./images/photos/5.jpg"
+                        alt="Place 5"
+                        class="place__photo"
+                      >
+                    </a>
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--desktop-5-12">
+                  <div class="place__photo-container place__photo-container--right">
+                    <a
+                      href="https://goo.gl/maps/jUJWjAFsmLsUbAch9"
+                      target="_blank"
+                      rel="noopener"
+                    >
+                      <img
+                        src="./images/photos/6.jpg"
+                        alt="Place 6"
+                        class="place__photo"
+                      >
+                    </a>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                href="https://goo.gl/maps/jUJWjAFsmLsUbAch9"
+                class="place__link"
+                target="_blank"
+                rel="noopener"
+              >
+                Faena House
+              </a>
+              / Miami Beach, FL 33140
+            </div>
+          </div>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--6-7">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section testimonials" id="testimonials">
```

Comment:
Good job on trying to add your own hover effects, but it may be better to stick with default ones, cause now text in testimonials overlaps on hover https://prnt.sc/1tx3go2 (((9 You may try to fix it, of course